gems/smithy/lib/smithy/views/client/client.rb

```diff
@@ -60,6 +60,22 @@ def protocols
           @protocols ||= @plan.welds.map(&:protocols).reduce({}, :merge)
         end
 
+        def waiters
+          waiters = Views::Client::Waiters.new(@plan).waiters
+          lines = ['{']
+          waiters.each_with_index do |waiter, i|
+            line = '  ' + waiter.name.underscore + ': Waiters::' + waiter.name + ','
```

Comment:
Use interpolation. Do you have RoboCop set up locally? I'm surprised it didn't complain here.